cmake/Modules/info.cmake

```diff
@@ -1,4 +1,4 @@
-get_directory_property(DEFINES COMPILE_DEFINITIONS)
+get_target_property(DEFINES OpenSCAD COMPILE_DEFINITIONS)
```

Comment:
Ah, nice. I missed that. Good to have that back, it has been quite useful when looking at CI build issues. Using target properties seems much better.